src/components/FileTreeNode/index.spec.tsx

```diff
@@ -216,4 +246,200 @@ describe(__filename, () => {
 
     expect(root.find(`.${styles.selected}`)).toHaveLength(0);
   });
+
+  it.each([
+    ['error', `.${styles.hasLinterErrors}`, 'times-circle'],
+    ['notice', `.${styles.hasLinterMessages}`, 'info-circle'],
+    ['warning', `.${styles.hasLinterWarnings}`, 'exclamation-triangle'],
+  ])('renders a file node that has linter %ss', (type, className, icon) => {
+    const message = {
+      ...fakeExternalLinterMessage,
+      file: 'manifest.json',
+      type: type as ExternalLinterMessage['type'],
+    };
+
+    const root = renderWithLinterMessage({
+      message,
+      treefoldRenderProps: {
+        isFolder: false,
+      },
+    });
+
+    expect(root.find(className)).toHaveLength(1);
+
+    const nodeIcons = root.find(`.${styles.nodeIcons}`);
+    expect(nodeIcons).toHaveLength(1);
+    expect(nodeIcons.find(FontAwesomeIcon)).toHaveProp('icon', icon);
+    expect(nodeIcons.find(FontAwesomeIcon).prop('title')).toMatch(
+      new RegExp(`file contains linter ${type}s`),
+    );
+  });
+
+  it.each([
+    ['error', `.${styles.hasLinterErrors}`, 'times-circle'],
+    ['notice', `.${styles.hasLinterMessages}`, 'info-circle'],
+    ['warning', `.${styles.hasLinterWarnings}`, 'exclamation-triangle'],
+  ])(
+    'renders a directory node that has linter %ss',
+    (type, className, icon) => {
+      const message = {
+        ...fakeExternalLinterMessage,
+        file: 'manifest.json',
```

Comment:
Could you make this more realistic for the test? Was this path supposed to be a directory?

Reply:
yep, thanks